test/tests/pbs_smoketest.py

```diff
@@ -1214,10 +1214,14 @@ def test_resource_create_delete(self):
         attr = {'Resources_available.foo': True}
         self.server.manager(MGR_CMD_SET, SERVER, attr,
                             id=self.server.shortname)
+
+        vnode_val = self.mom.shortname
+        if self.mom.is_cpuset_mom():
+            vnode_val = self.server.status(NODE)[1]['id']
```

Comment:
Just being robust here, might want to check first if indeed, there's a <node_name>[0] entry before assigning it, otherwise, we would get an exception. Sure if it's a cpuset mom, we would expect it, but better be safe here.

Reply:
or at least do a if len(status_val) > 1